concordium-std/tests/derive-deletable/success-ident-param.rs

```diff
@@ -0,0 +1,9 @@
+use concordium_std::{Deletable, HasStateApi, StateMap};
```

Comment:
Please document the intent of this test as a module doc comment, i.e., `//!`.

Reply:
Added